### ISSUE TITLE
`\n` not detected properly

### ISSUE THREAD

**Issue by gamethis** [adrienverge/yamllint#296]
when linting a string like a one of the following yamllint gives a false error
error    wrong new line character: expected \n  (new-lines)

sample false positives:

```
- name: update setting
  win_regedit:
    path: "HKLM:\\Software\\Policies\\Microsoft\\Windows\\Network Connections"
    name: "NC_AllowNetBridge_NLA"
    data: "0"
    type: dword

- name: update setting
  win_regedit:
    path: HKLM:\Software\Policies\Microsoft\Windows\Network Connections
    name: "NC_AllowNetBridge_NLA"
    data: "0"
    type: dword
```

The problem occurs with either syntax above

**Comment by adrienverge** [comment]
Hello,

I copied-pasted your example and I don't encounter the false positive you describe.

Have you considered that the yamllint error could be true? Could you inspect your file byte by byte: maybe it really contains `\r` chars?

**Comment by gamethis** [comment]
so it ended up being detected properly thank you.
